packages/components/src/RecurringSelect/RecurringSelect.mdx

```diff
@@ -54,12 +72,15 @@ import { RecurringSelect } from "@jobber/components/RecurringSelect";
   }}
 </Playground>
 
-## Props
-
-<Props of={RecurringSelect} />
-
 ## Accessibility
 
 The `RecurringSelect` component is entirely usable by keyboard.
 
-All selections can be done and undone with tabs, arrows and the space bar.
+All selections can be done and undone with tabs, arrows and the space bar. The "visual" 
```

Comment:
I realize that it was most likely me who wrote the "arrows" bit here originally, but it's not strictly true sadly

while some parts of the UI such as the dropdown or radios can be navigated with arrows, the main calendar portions such as Week/Month cannot be accessed with arrows - only tabs & shift-tabs

I feel like it would be preferable to remove that, so as to prevent disappointment and anger when someone tries to use arrows and decides we are lying liars who lie